astropy/io/fits/util.py

```diff
@@ -380,14 +380,9 @@ def isfile(f):
 def fileobj_open(filename, mode):
     """
     A wrapper around the `open()` builtin.
-
-    This exists because `open()` returns an `io.BufferedReader` by default.
-    This is bad, because `io.BufferedReader` doesn't support random access,
-    which we need in some cases.  We must call open with buffering=0 to get
-    a raw random-access file reader.
     """
 
-    return open(filename, mode, buffering=0)
```

Comment:
Why does this feel like it is undoing something we put in on purpose?

Reply:
Well, git blame says I am crazy. Maybe I am confused because I saw thing change over at that other PR.

Reply:
I think it had been put in on purpose, but is no longer required now (except on Windows), or has been overreaching – see https://github.com/astropy/astropy/pull/11955#issuecomment-883236841.

Reply:
> no longer required now (except on Windows)

I didn't follow the original discussion so maybe this has been discussed, but if it is still required on Windows, is it okay to just remove like that?